packages/provider.databind/spec/dataBindProviderBehaviors.ts

```diff
@@ -10,26 +10,24 @@
 */
```

Comment:
Check it with 'hide whitespaces'. The file was not well formatted, so I did an autoformat.